### PR TITLE
Decode HTML entities in descriptions

### LINKED ISSUE #17: Fix HTML character encoding in descriptions
```
properties
 description: 'Issuer&apos;s Common Name' },
```

### PR DESCRIPTION
**Pull request by zeke**
Fixes #17

### CODE REVIEW

**Comment by MarshallOfSound** [comment]
Linter oh glorious linter

```bash
test/index.js:138:60: Missing space before function parentheses.
```

**Comment by zeke** [comment]
I've been slow to adapt to this rule. 🐌

**Comment by MarshallOfSound** [comment]
Heh, not as bad as me.  I'm constantly switching between `eslint-config-airbnb` on all my projects and `standard` on Electron projects.  They basically disagree on every single rule 😆

**Comment by zeke** [comment]
This is ready for 👀

File: test/index.js [hunk 1]
```diff
@@ -135,8 +135,11 @@ describe('APIs', function () {
       expect(keys).to.not.include('required')
     })
 
-    they('do not contain HTML encoded characters', function() {
-      // expect(JSON.stringify(apis)).to.not.include('&apos;')
+    they('do not contain HTML encoded characters', function () {
+
```
Comment (MarshallOfSound) [inline test/index.js]: standard says nono to this blank line

**Pull request merged**